### ISSUE TITLE
homestead blockchain not syncing correctly

### ISSUE THREAD

**Issue by jayraj9700** [ethcore/parity#3969]
Installed latest version of parity on windows 10, it runs well however, when i monitor the best block against ethstats.net - i notice my parity is running behind like 10-20 blocks most of the time. Then after few minutes parity sync's multiple blocks together and stays up-to-date. Again after few minutes the sync status falls behind by 10-20 blocks. 

I checked my system time is exact, even peer counts is healthy (18-20) & also deleted the node.js file and restarted parity, still the same problem. Have attached the log below after running parity with  -l sync=trace option. am running version - Parity v1.4.6-beta 

[parity_log.txt](https://github.com/ethcore/parity/files/672235/parity_log.txt)

**Comment by gavofyork** [comment]
I'm seeing similar performance on a cleanly built master, too.

```
Gavs-MBP:parity gav$ ./target/release/parity -l sync=trace 2> ~/sync.trace
2016-12-28 12:18:10 UTC main INFO parity::run  Starting Parity/v1.5.0-unstable-4e51f93-20161227/x86_64-macos/rustc1.14.0
2016-12-28 12:18:10 UTC main INFO parity::run  State DB configuration: fast
2016-12-28 12:18:10 UTC main INFO parity::run  Operating mode: active
2016-12-28 12:18:10 UTC main INFO ethcore::service  Configured for Frontier/Homestead using Ethash engine
2016-12-28 12:18:15 UTC  WARN signer  Received old authentication request. (1482927495 vs 1482927485)
2016-12-28 12:18:15 UTC  INFO signer  Unauthorized connection to Signer API blocked.
2016-12-28 12:18:17 UTC IO Worker #3 INFO network  Public node URL: enode://686d70b43a17a5de21e7736dc1aad5d5cc4c75bdf84b20ba599f8bd10d4bea4661f91f5bc76b3fb4c7782f51b974e41b20ff914c74a637ba9ec5d480dc6c4b48@192.168.1.16:30303
2016-12-28 12:18:27 UTC IO Worker #0 INFO import  Syncing #2891335 6a1e…6195      0 blk/s    2 tx/s   0 Mgas/s       0+    0 Qed   #2891335    3/ 3/25 peers     57 MiB db  233 KiB chain  0 bytes queue  497 KiB sync
2016-12-28 12:18:27 UTC Verifier #3 INFO import  Imported #2891336 f4cf…a3ea (0 txs, 0.00 Mgas, 1.17 ms, 0.52 KiB) + another 2 block(s) containing 36 tx(s)
2016-12-28 12:18:37 UTC Verifier #2 INFO import  Imported #2891338 ae8e…675f (0 txs, 0.00 Mgas, 0.60 ms, 0.52 KiB) + another 1 block(s) containing 0 tx(s)
2016-12-28 12:18:57 UTC IO Worker #0 INFO import      #242023    5/ 5/25 peers     57 MiB db    1 MiB chain  0 bytes queue  949 KiB sync
2016-12-28 12:19:27 UTC IO Worker #3 INFO import      #244817    2/ 2/25 peers     57 MiB db    3 MiB chain  0 bytes queue  985 KiB sync
2016-12-28 12:19:52 UTC IO Worker #1 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891338    1/ 1/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:19:57 UTC IO Worker #3 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891338    1/ 1/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:20:17 UTC IO Worker #3 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891338    3/ 3/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:20:22 UTC IO Worker #1 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891338    3/ 3/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:20:27 UTC IO Worker #2 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891338    2/ 2/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:20:42 UTC IO Worker #3 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891337    3/ 3/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:20:47 UTC IO Worker #0 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891337    3/ 3/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:20:52 UTC IO Worker #2 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891337    3/ 3/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:21:07 UTC IO Worker #1 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891337    2/ 3/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:21:12 UTC IO Worker #1 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891337    1/ 2/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
2016-12-28 12:21:17 UTC IO Worker #0 INFO import  Syncing #2891338 ae8e…675f      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2891337    3/ 3/25 peers     57 MiB db    3 MiB chain  0 bytes queue    1 MiB sync
```

[sync.trace.txt](https://github.com/ethcore/parity/files/675543/sync.trace.txt)

Decent internet connection, recently warp-synced (still catching up on old blocks).

**Comment by gavofyork** [comment]
more logs:
[sync.trace.txt](https://github.com/ethcore/parity/files/675590/sync.trace.txt)

**Comment by arkpar** [comment]
The original issue reported by @jayraj9700 should be fixed in 1.4.8. @jayraj9700 Could you please check if it works for you in the latest version?

**Comment by arkpar** [comment]
Assuming fixed. Please reopen or file a new issue if this still happens.
